adala/runtimes/_openai.py

```diff
@@ -167,7 +167,10 @@ class OpenAIChatRuntime(Runtime):
 
     @computed_field
     def _client(self) -> OpenAI:
-        return OpenAI(api_key=self.openai_api_key)
+        return OpenAI(
+            api_key=self.openai_api_key,
+            base_url=DEFAULT_CREATE_COMPLETION_URL
```

Comment:
Does the base URL refer to the completion endpoint but not something like `https://api.openai.com/v1` ?